intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/plugin/PluginV2Dependency.kt

```diff
@@ -4,12 +4,10 @@
 
 package com.jetbrains.plugin.structure.intellij.plugin
 
-data class PluginV2Dependency(private val id: String) : PluginDependency {
-  override fun getId() = id
+data class PluginV2Dependency(override val id: String, override var isOptional: Boolean = false) : PluginDependency {
+  override val isModule = id.startsWith(INTELLIJ_MODULE_PREFIX)
```

Comment:
I'm not sure that is always the case

Reply:
I have seen this issue and researched it. This is a pickup from previous implementation with unknown semantics.